_posts/0000-01-02-parkerbxyz.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is fine. lol
 
 ![poptocat](https://octodex.github.com/images/poptocat.png)
```

Comment:
If you want to change the picture, you can grab it from [here](https://octodex.github.com)